example_model/requirements.txt

```diff
@@ -1,3 +1,5 @@
+telesto-base
```

Comment:
As I have indicated in my comment at your PR for `telesto-base`, this package should be already installed in the base image.

Reply:
I think it is. Also fixed.